libreCaptcha.php

```diff
@@ -45,52 +42,64 @@ function libre_captcha_id($level = "easy", $media = "image", $input_type = "text
 		return -1;
 }
 
-function libre_get_media($id){
-	global $lc_url;
+function libre_get_media($id, $lc_url){
 	$media_data = array('id' => $id);
 	$image = libreAPI($lc_url.'/v1/media/', json_encode($media_data));
 	return $image;
 }
 
-function libre_get_answer($id, $answer){
-	global $lc_url;
+function libre_get_answer($id, $answer, $lc_url){
 	$answer_data = array('id' => $id, 'answer' => $answer);
 	$get_answer = libreAPI($lc_url.'/v1/answer/', json_encode($answer_data));
 	$answer = json_decode($get_answer, TRUE);
 	return $answer['result'];
 }
 
 function libre_set_lc_url(){
```

Comment:
Now that it doesn't set any global variable, this function can be renamed to `libre_get_lc_url`